theseus/geometry/point_types.py

```diff
@@ -155,7 +169,21 @@ def z(self) -> torch.Tensor:
         return self[:, 2]
 
     @staticmethod
-    def exp_map(tangent_vector: torch.Tensor) -> "Point3":
+    def exp_map(
+        tangent_vector: torch.Tensor, jacobians: Optional[List[torch.Tensor]] = None
+    ) -> "Point3":
+        if tangent_vector.ndim != 2 or tangent_vector.shape[1] != 3:
+            raise ValueError("Tangent vectors of Point3 should be 3-D vectors.")
+
+        if jacobians is not None:
+            shape = tangent_vector.shape
+            Point2._check_jacobians_list(jacobians)
```

Comment:
Should it be `Point3` here? BTW, better to add a common method here for `Point2` and `Point3`, parameterized by the class/dof, since the logic is pretty much the same.

Reply:
Any way to just call the Vector.exp_map with the dof instead of a common Point2/Point3 function?

Reply:
The reason why I use `Point3` is to pass the `precommit` checks. I forgot the exact issues if using `Vector`.

Reply:
Ah, I see. I will fix this. But there should be no differences since both of them call `LieGroup._check_jacobians_list`.